tests/foreman/api/test_computeresource_libvirt.py

```diff
@@ -63,88 +50,86 @@ def test_positive_create_with_name(setup, name):
-def test_positive_create_with_description(setup, description):
-    """Create compute resources with different descriptions
+@pytest.mark.parametrize('name', **parametrized(valid_data_list()))
+def test_positive_create_with_name_description(
+    name, request, module_target_sat, module_org, module_location
+):
+    """Create compute resources with different names and descriptions
 
-    :id: 1fa5b35d-ee47-452b-bb5f-4a4ca321f992
+    :id: 1e545c56-2f53-44c1-a17e-38c83f8fe0c1
 
-    :expectedresults: Compute resources are created with expected
-        descriptions
+    :expectedresults: Compute resources are created with expected names and descriptions
 
     :CaseImportance: Critical
 
     :CaseLevel: Component
 
     :parametrized: yes
     """
-    compresource = entities.LibvirtComputeResource(
-        description=description,
-        location=[setup.loc],
-        organization=[setup.org],
-        url=setup.current_libvirt_url,
-    ).create()
-    assert compresource.description == description
-
-
-@pytest.mark.tier2
-@pytest.mark.parametrize('display_type', ['spice', 'vnc'])
-def test_positive_create_libvirt_with_display_type(setup, display_type):
-    """Create a libvirt compute resources with different values of
-    'display_type' parameter
-
-    :id: 76380f31-e217-4ff1-ac6b-20f41e59f133
-
-    :expectedresults: Compute resources are created with expected
-        display_type value
-
-    :CaseImportance: High
-
-    :CaseLevel: Component
-
-    :parametrized: yes
-    """
-    compresource = entities.LibvirtComputeResource(
-        display_type=display_type,
-        location=[setup.loc],
-        organization=[setup.org],
-        url=setup.current_libvirt_url,
+    compresource = module_target_sat.api.LibvirtComputeResource(
+        name=name,
+        description=name,
+        organization=[module_org],
+        location=[module_location],
+        url=LIBVIRT_URL,
     ).create()
-    assert compresource.display_type == display_type
-
-
-@pytest.mark.tier1
-def test_positive_create_with_provider(setup):
-    """Create compute resources with different providers. Testing only
-    Libvirt and Docker as other providers require valid credentials
-
-    :id: f61c66c9-15f8-4b00-9e53-7ebfb09397cc
-
-    :expectedresults: Compute resources are created with expected providers
-
-    :CaseImportance: Critical
-
-    :CaseLevel: Component
-    """
-    entity = entities.LibvirtComputeResource()
-    entity.location = [setup.loc]
-    entity.organization = [setup.org]
-    result = entity.create()
-    assert result.provider == entity.provider
+    assert compresource.name == name
+    assert compresource.description == name
+    request.addfinalizer(compresource.delete)
 
 
 @pytest.mark.tier2
-def test_positive_create_with_locs(setup):
+def test_positive_create_with_orgs_and_locs(request, module_target_sat):
     """Create a compute resource with multiple locations
```

Comment:
Can you update the docstring here and in expectedresults to include orgs also.

Reply:
Updated